src/components/Home/Home.tsx

```diff
@@ -8,21 +10,11 @@ const Home: React.FunctionComponent = () => {
   const [value, setValue] = useState('');
   const [isValid, setIsValid] = useState(false);
 
-  const inputOnChange = (event: React.ChangeEvent<HTMLInputElement>): void => {
```

Comment:
This should probably cause conflicts with master since the same change already exists currently on master. You should drop these changes and update your branch with the current version of master